deploy/template.yaml

```diff
@@ -59,19 +59,29 @@ Mappings:
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
       ISSUER: 'https://review-o.dev.account.gov.uk'
+      CLIENTS: '[{"jwksEndpoint":"https://ipvstub.review-o.dev.account.gov.uk/.well-known/jwks.json","clientId":"5C584572","redirectUri":"https://ipvstub.review-o.dev.account.gov.uk/redirect"}]'
```

Comment:
Change the jwksEndpoint here to 
https://0qtxg80z58.execute-api.eu-west-2.amazonaws.com/dev/.well-known/jwks.json and redirectUri to https://ipvstub.review-c.dev.account.gov.uk/redirect
Just to unblock for testing

Reply:
Stub needs a custom Domain attached in APIGw @maruf-ahmedd  @CoralSmith 
then jwksEndpoint here will need updating @hemavana 
REDIRECT_URI value needs changing in the template.yaml in ipv-stub